netbox/utilities/forms/fields/csv.py

```diff
@@ -57,6 +57,11 @@ class CSVModelChoiceField(forms.ModelChoiceField):
         'invalid_choice': _('Object not found: %(value)s'),
     }
 
+    def __init__(self, conditional=False, *args, **kwargs):
+        # Used to display tags for fields that are conditionally required
+        self.conditional = conditional
+        super().__init__(*args, **kwargs)
+
```

Comment:
I am not myself prepared to make a decision on whether this is something we should consider introducing.

That said, I think that if this is something we want to persue further, there should be a FR for it and a related PR as I am sure there is more then just this.

For example, it marks it as a conditionally required value but do we want to enrich that further by telling the end-user which fields are part of this conditional group?